doc/metadata_routing.rst

```diff
@@ -91,13 +91,13 @@ since ``sample_weigh`` was not requested by any of its underlying objects.
 Weighted scoring and unweighted fitting
 ---------------------------------------
 
-When passing metadata such as ``sample_weight`` around, all scikit-learn
-estimators require weights to be either explicitly requested or not requested
-(i.e. ``True`` or ``False``) when used in another router such as a
-:class:`~pipeline.Pipeline` or a ``*GridSearchCV``. To perform an unweighted
-fit, we need to configure :class:`~linear_model.LogisticRegressionCV` to not
-request sample weights, so that :func:`~model_selection.cross_validate` does
-not pass the weights along::
+When passing metadata such as ``sample_weight`` around, all ``sample_weight``
+:term:`consumer`s require weights to be either explicitly requested or not
```

Comment:
Based on the [CI errors](https://output.circle-artifacts.com/output/job/665df8b6-f6f2-448b-bc7e-bf58e0f8f440/artifacts/0/doc/_changed.html), I suspect we need to write:

```
:term:`consumers <consumer>`
```